app/src/main/res/values/strings.xml

```diff
@@ -3,9 +3,9 @@
 
     <string name="title_section1">Discover</string>
     <string name="title_section2">Make</string>
-    <string name="title_activity_map">Map</string>
     <string name="title_activity_project">Project</string>
-    <string name="title_activity_editor">Options</string>
+    <string name="title_activity_page">Page</string>
+    <string name="title_activity_element">Options</string>
```

Comment:
Do these two strings still make sense? (I got confused seeing the word options while editing my element, because I wasn't in an options menu, I was editing my content)